app/code/Magento/CatalogGraphQl/Model/Resolver/Products/Query/Filter.php

```diff
@@ -82,6 +84,11 @@ public function getResult(
         return $this->searchResultFactory->create($products->getTotalCount(), $productArray);
     }
 
+    public function getLayerType(): string
+    {
+        return \Magento\Catalog\Model\Layer\Resolver::CATALOG_LAYER_CATEGORY;
```

Comment:
@pmclain does it sound right to you that this layer is called `Category layer`? Is there better name?

Reply:
I'm not a huge fan of the name but have grown complacent with it over the years. `CATALOG_LAYER_DEFAULT` would be my personal preference. Would this need to be done as an alias for BC? Something like this:
```php
class Resolver
{
    const CATALOG_LAYER_CATEGORY = 'category';
    const CATALOG_LAYER_SEARCH = 'search';
    const CATALOG_LAYER_DEFAULT = static::CATALOG_LAYER_CATEGORY;
...
```

Reply:
That is what I was thinking about, please change the constant along with its value.

We do not have any BC requirements for GraphQL classes in 2.3.0 since it was "developer preview". We only guarantee GraphQL schema for now.

Reply:
@paliarush Just to confirm, that constant was not introduced by graphql and predates the first stable m2 release https://github.com/magento/magento2/blame/2.3-develop/app/code/Magento/Catalog/Model/Layer/Resolver.php#L11 I don't think it is safe to change the value.

Reply:
You are right, I thought it was defined in CatalogGraphQL module. Let's keep it as is then. Thanks!